libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/market-roles/market-role-service.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { EicFunction } from '@energinet-datahub/dh/shared/domain';
+
+export class MarketRoleService {
```

Comment:
suggestion: to make it a "real" Angular service, add an `@Injectable()` decorator at the top of the class (similar to "dh-market-participant-edit-actor-data-access-api.store.ts#75").